### ISSUE TITLE
Support for TabPanel to have better keyboard navigation control 

### ISSUE THREAD

**Issue by vini-gupta** [microsoft/fluent-ui-react#1170]
## Feature Request
Support for TabPanel to have better keyboard navigation control from tab header to tab content

### Problem description
Currently when we are using Menu as a tablist and rendering the corresponding panel as a layout we are running into issues to move keyboard focus across these two components. Ideally a tab panel should be able to move keyboard focus from tab header to the underlying tab panel as soon as user issues a keyboard event.

**Comment by hughreeling** [comment]
Please provide accessibility specification for this.

**Comment by hughreeling** [comment]
After discussions with Reach team it was concluded that this was not needed.   Please close.
